apps/openmw/mwscript/miscextensions.cpp

```diff
@@ -469,6 +471,9 @@ namespace MWScript
                     std::string gem = runtime.getStringLiteral (runtime[0].mInteger);
                     runtime.pop();
 
+                    if (!ptr.getClass().hasInventoryStore(ptr))
```

Comment:
According to my testing, this commands can be applied for any creature in Morrowind, not just for ones with inventory.
Also Morrowind seems to spawn items near player, not near target actor.

From what I can tell, Drop uses the same approach to place items as a "player->placeatme" in Morrowind, but I am not sure if it worth to replicate.

Reply:
>this commands can be applied for any creature in Morrowind, not just for ones with inventory.

Ok.
>Also Morrowind seems to spawn items near player, not near target actor.

Vanilla bug.
>but I am not sure if it worth to replicate.

Nuances, nuances. It's not the same as PlaceAt because there isn't a new reference generated in Morrowind if there *is* an instance of an item in the inventory. It could copy the first instance of the item in theory, but the understanding isn't clear enough.